database/migrations/2020_01_08_043027_create_blogs_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
```

Comment:
This is part of big blog task (and it not affect other task) , please create big branch blog and do pull request merge each part to that big branch before request merge into develop

Reply:
Thanks, I will fix.